@xen-orchestra/lite/src/components/ui/UiStatusPill.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiIcon :icon class="icon" :class="props.state" />
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@/components/ui/icon/UiIcon.vue'
+import { computed } from 'vue'
+import { faCheckCircle, faCircleMinus, faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import type { Color } from '@/types'
+
+const props = defineProps<{
+  state: Color
+}>()
+
+const icon = computed(() => {
+  switch (props.state) {
+    case 'error':
+      return faCircleXmark
+    case 'warning':
+      return faCircleMinus
+    case 'success':
+      return faCheckCircle
+    default:
+      return faCircleXmark
+  }
+})
+</script>
+
+<style scoped lang="postcss">
+.icon {
```

Comment:
According to the design system, the `font-size` should be `10px`.
```suggestion
.status-pill {
  font-size: 1rem;
}
```